lib/OpenQA/Worker/Common.pm

```diff
@@ -467,7 +472,7 @@ sub register_worker {
         $worker_caps->{worker_class} = 'qemu_' . $worker_caps->{cpu_arch};
     }
 
-    log_info("registering worker with openQA $host...");
+    log_info("registering worker [@{[INTERFACE_VERSION]}] with openQA $host...");
```

Comment:
I would change the message to: _Registering worker '$worker_hostname' with openQA $host using protocol version [@{[INTERFACE_VERSION]}]_